lib/admin.rb

```diff
@@ -0,0 +1,125 @@
+require 'time'
+require 'date'
+require 'pry'
+
+require_relative 'room'
+require_relative 'reservation'
+
+module Hotel
+  class Admin
+    attr_reader :rooms, :reservations
+
+    def initialize
+      @rooms = all_rooms
+      @reservations = []
+      @blocks = []
+    end
+
+    def all_rooms
+      room_array = []
+      x = 1
+      20.times do
+        room_array << Hotel::Room.new(x)
```

Comment:
Note that when you work from within the module, you shouldn't need to preface everything with `Hotel::`